renderer/components/ui/AutocompleteSelector.tsx

```diff
@@ -94,6 +98,9 @@ export default function AutocompleteSelector({
     if (selectedItem.startsWith("TT:: ")) {
       setSelectedItem((prev) => prev.slice(5));
     }
+    if (isNewCheck) {
+      setIsNew(selectedItem && !availableItems.includes(selectedItem));
+    }
```

Comment:
we need to make sure availableItems will never be undefined or use optiaonal chaining